<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EMMN</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,6">
    <item row="6" column="0" colspan="2">
     <layout class="QHBoxLayout" name="trackingModeLayout">
      <item>
       <widget class="QRadioButton" name="noneModeRadio">
        <property name="text">
         <string>Repouso</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="satModeRadio">
        <property name="text">
         <string>Satélites</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sunModeRadio">
        <property name="text">
         <string>Sol</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Satélites</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="satellitesView">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showAllPassesButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Mostrar todas as passagens</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="satelliteGroupBox">
        <property name="title">
         <string>Satélite</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Azimute</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Elevação</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="satEle">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="satAz">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Próx. passagem</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="satNextPass">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="selectedSatOptions">
        <item>
         <widget class="QPushButton" name="removeTrackerButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Deletar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moveTrackerDownButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>▼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moveTrackerUpButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>▲</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addTrackerButton">
        <property name="text">
         <string>Adicionar novo satélite</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="passesInfoLayout">
      <item>
       <widget class="QLabel" name="passesViewLabel">
        <property name="text">
         <string>Passagens (próximos sete dias)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="passesView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
        <item>
         <widget class="QGroupBox" name="nextPassInfoGroupBox">
          <property name="title">
           <string>Próxima passagem</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Tempo restante</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="nextPassCountdownLabel">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Satélite</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="nextPassSatLabel">
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="antennaInfoGroupBox">
          <property name="title">
           <string>Antena</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Azimute (mec)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="azMecLabel">
               <property name="text">
                <string>---</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Elevação</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="eleLabel">
               <property name="text">
                <string>---</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <item row="0" column="1">
              <widget class="QLabel" name="azGeoLabel">
               <property name="text">
                <string>---</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Azimute (geo)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Power</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="powerLabel">
               <property name="text">
                <string>---</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Offset</string>
          </property>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Azimute</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="azOffset">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>15</height>
              </size>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QGroupBox" name="nextPassesGroupBox">
      <property name="title">
       <string>Próximas passagens</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>915</width>
            <height>69</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="NextPassesView" name="nextPassesView" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="actionUpdateTLEs"/>
    <addaction name="actionOpenLog"/>
    <addaction name="actionDebugar"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionSobre"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionDesfazer"/>
    <addaction name="actionRefazer"/>
   </widget>
   <widget class="QMenu" name="menuAntena">
    <property name="title">
     <string>Antena</string>
    </property>
    <addaction name="actionManualControl"/>
    <addaction name="actionSendPower"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuAntena"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCarregar_efem_ride">
   <property name="text">
    <string>Carregar efemérides</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Configurações</string>
   </property>
  </action>
  <action name="actionDebugar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debugar</string>
   </property>
  </action>
  <action name="actionManualControl">
   <property name="text">
    <string>Controle manual</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manual do usuário</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionDesfazer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Desfazer</string>
   </property>
  </action>
  <action name="actionRefazer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Refazer</string>
   </property>
  </action>
  <action name="actionUpdateTLEs">
   <property name="text">
    <string>Atualizar TLEs</string>
   </property>
  </action>
  <action name="actionSendPower">
   <property name="text">
    <string>Enviar Power</string>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>Abrir log</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NextPassesView</class>
   <extends>QWidget</extends>
   <header>nextpassesview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>satellitesView</tabstop>
  <tabstop>showAllPassesButton</tabstop>
  <tabstop>removeTrackerButton</tabstop>
  <tabstop>moveTrackerDownButton</tabstop>
  <tabstop>moveTrackerUpButton</tabstop>
  <tabstop>addTrackerButton</tabstop>
  <tabstop>passesView</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
